src/figuresClasses.ts

```diff
@@ -1,19 +1,85 @@
-export interface Figure {
-
+enum Shapes {
+  Triangle = 'triangle',
+  Circle = 'circle',
+  Rectangle = 'rectangle',
 }
 
-export class Triangle {
+enum Colors {
+  'red',
+  'green',
+  'blue',
+}
 
+export interface Figure {
+  shape: Shapes;
+  color: Colors;
+  getArea(): number;
 }
 
-export class Circle {
+export class Triangle implements Figure {
+  shape: Shapes = Shapes.Triangle;
```

Comment:
Не обов`язково тут писати цей тип, TS сам його підтягне))
```suggestion
  shape = Shapes.Triangle;
```